provider/datagen/src/tests/make_testdata.rs

```diff
@@ -58,7 +58,7 @@ fn generate_json_and_verify_postcard() {
     });
 
     DatagenDriver::new()
-        .with_keys(icu_datagen::all_keys())
+        .with_keys(crate::all_keys())
```

Comment:
cannot comment on line 32: you can now use `new_testing`, which is `cfg(test)` as well

Reply:
Done